src/pages/en/core-concepts/astro-components.md

```diff
@@ -107,26 +107,26 @@ const myFavoritePokemon = [/* ... */];
 <p set:html={rawHTMLString} />
 ```
 
-### Dynamic JSX Expressions
+### JSX Expressions
 
-Astro components can define local variables inside of the frontmatter component script. Any script variables are then automatically available in the component's HTML template below.
+You can can define local JavaScript variables inside of the frontmatter component script within an Astro component. You can then inject these variables into the HTML using JSX expressions!
 
-#### Dynamic Values
+#### Variables
 
-These local variables can be used in curly braces to pass values to be used as HTML output:
+Local variables can be added into the HTML using the using curly braces syntax:
 
 ```astro
 ---
 const name = "Astro";
 ---
 <div>
-  <h1>Hello {name}!</h1>
+  <h1>Hello {name}!</h1> // Outputs <h1>Hello Astro!</h1>
 </div>
 ```
 
-#### Dynamic Attributes
+#### Attributes
```

Comment:
```suggestion
#### Dynamic Attributes
```
We have some discussions going on specifically re: the use of the words "dynamic" and "interactive"... so I'm not going to approve a change this just yet until some other decisions are made. But, I've made a note of this!